connect-base/0.5/Dockerfile

```diff
@@ -28,5 +28,15 @@ RUN mkdir $KAFKA_CONNECT_PLUGINS_DIR;
 # on the classpath, which results in arcane NoSuchMethodError exceptions.
 #
 
+RUN cd $KAFKA_HOME/libs &&\
```

Comment:
Do you know whether putting them to libs will be still the way to go as of 0.11? Nothing to act on now, but may be worth double-checking when doing the upgrade to 0.11 in DBZ 0.6.